server/requirements.txt

```diff
@@ -8,3 +8,4 @@ flask
 gunicorn==19.3.0
```

Comment:
Are we still using it?

Reply:
Not now, you are right! I don't run the app with `gunicorn` anymore. It has to be used for production, but I think we can get rid of it for this project